policy/modules/contrib/qgs.fc

```diff
@@ -0,0 +1,8 @@
+/etc/qgs.conf --             gen_context(system_u:object_r:qgs_etc_t,s0)
+
+/etc/sgx_default_qcnl.conf  gen_context(system_u:object_r:sgx_qcnl_etc_t,s0)
```

Comment:
Use "--" for a plain file.

Reply:
Can't find a good one now, but there are
https://github.com/OwlCyberDefense/refpolicy/wiki/GettingStarted#module-fc-policy-modulefcpolicy
and
https://access.redhat.com/articles/6999267#file-context-13

For the blank file type: applies to all types if not specified.